packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -534,6 +562,20 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
     //                          HELPERS                           //
     ////////////////////////////////////////////////////////////////
 
+    /// @notice Pays out the bond of a claim to a given recipient.
+    /// @param _recipient The recipient of the bond.
+    /// @param _bonded The claim to pay out the bond of.
+    /// @return success_ Whether or not the transfer was successful.
+    function _payBond(address _recipient, ClaimData storage _bonded) internal returns (bool success_) {
+        // Set all bits in the bond value to indicate that the bond has been paid out.
+        uint256 bond = _bonded.bond;
+        if (bond == type(uint128).max) revert ClaimAlreadyResolved();
+        _bonded.bond = type(uint128).max;
+
+        // Pay out the bond to the recipient.
+        (success_,) = address(_recipient).call{ value: bond }(hex"");
+    }
```

Comment:
I'm much more in favor of using the pull vs push pattern here. This will undoubtedly be a security concern in any given audit.

Reply:
Definitely something to discuss - I think that the push pattern is pretty safe here, and the solvency invariant is pretty clear to test. Ticketed out for further discussion https://github.com/ethereum-optimism/client-pod/issues/429

Reply:
Swapped to pull